### PR TITLE
feat(frontend): implement service for communicating with Blockchain API

### PR DESCRIPTION
**Pull request by DenysKarmazynDFINITY**
# Motivation

The goal is to implement service for communicating with Blockchain API. Initially, it will be used for fetching uncertified BTC transactions.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -8,6 +8,7 @@ VITE_OISY_URL=https://oisy.com
 VITE_ERC20_ICP_SYMBOL=ckICP
 VITE_ERC20_ICP_REPO_URL=https://github.com/dfinity/erc20-icp
 VITE_COINGECKO_API_URL=https://api.coingecko.com/api/v3
+VITE_BLOCKCHAIN_API_URL=https://blockchain.info
```
Comment (peterpeterparker) [inline .env.example:11]: That's correct? Their api endpoint is the same URL as their website?

**Review by peterpeterparker — APPROVED** [review]
LGMT, thanks

Code looks good, so already approving but, checkout the comment about the env file please before merge.

File: .env.example (hunk 1 above)
Comment (DenysKarmazynDFINITY) [reply .env.example:11]: Surprisingly, yes, it's the same URL.

Comment (peterpeterparker) [reply .env.example:11]: Weird there they say it's `api.blockchain.com` 🤷‍♂️
https://api.blockchain.com/v3/#authentication

Maybe I'm looking at the wrong documentation. Thanks for the feedback.

**Pull request merged**